css/styles.css

```diff
@@ -386,3 +412,181 @@ button.form_btn:hover {
   grid-template-rows: 25% 25% 25% 25%;
   row-gap: 20px;
 }
+
+.contenido-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.contenido-lenguajes {
+  display: flex;
+  flex-direction: column;
+}
+
+/*      Desktop Style     */
+
+@media (min-width: 768px) {
+  .portfolio {
+    display: flex;
+    display: none;
+  }
+
+  .portfolio1 {
+    display: flex;
+    width: auto;
+  }
+
+  .card-reverse {
+    display: flex;
+    flex-direction: row-reverse;
+    flex-wrap: nowrap;
+    justify-content: space-around;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  .project-detail {
+    width: 569px;
+  }
+
+  .contenido1 {
+    display: flex;
+    width: 40%;
+    height: auto;
+    flex-direction: column;
+    margin-left: 100px;
+  }
+
+  .about-me {
+    margin-top: 30px;
+  }
+
+  .about {
+    width: 500px;
+  }
+
+  .contenido-mobil {
+    display: none;
+  }
+
+  .contenido-desktop {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    width: auto;
+  }
+
+  .contenido-lenguajes {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .form {
+    display: flex;
+    background-image: url(../img/contact-background-big.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: left;
+    align-items: center;
+  }
+
+  .form-control {
+    display: flex;
+    width: 569px;
+  }
+
+  .form-p {
+    display: flex;
+    align-content: center;
+  }
```

Comment:
[OPTIONAL] Great job implementing the changes 🎉💯 I suggest adding `text-align: center` here to center the paragraph and to look similar to the Figma design 👌